pkg/crc/machine/bundle/repository.go

```diff
@@ -83,6 +83,11 @@ func (repo *Repository) Use(bundleName string) (*CrcBundleInfo, error) {
 func (bundle *CrcBundleInfo) createSymlinkOrCopyOpenShiftClient(ocBinDir string) error {
 	ocInBundle := bundle.resolvePath(constants.OcExecutableName)
 	ocInBinDir := filepath.Join(ocBinDir, constants.OcExecutableName)
+	if info, err := os.Stat(ocBinDir); err == nil && !info.IsDir() {
```

Comment:
Can you add a comment which says this is needed when upgrading from crc versions anterior to commit `1.11.0~5` ?

Reply:
done